components/org.wso2.carbon.identity.verification.onfido.api/org.wso2.carbon.identity.verification.onfido.api.common/pom.xml

```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) 2023, WSO2 LLC. (http://www.wso2.com).
```

Comment:
We should use 2024 right?

Reply:
Resolved via https://github.com/wso2-extensions/identity-verification-onfido/pull/12/commits/6f73cff0b813bfcfffbd6cfbe9d730df1f5fe3ed